nni/nas/hub/pytorch/autoformer.py

```diff
@@ -235,13 +235,21 @@ class MixedClsToken(MixedOperation, ClsToken):
     def super_init_argument(self, name: str, value_choice: ValueChoiceX):
         return max(traverse_all_options(value_choice))
 
-    def forward_with_args(self, embed_dim,
-                        inputs: torch.Tensor) -> torch.Tensor:
+    def slice_param(self, embed_dim, **kwargs) -> Any:
         embed_dim_ = _W(embed_dim)
         cls_token = _S(self.cls_token)[..., :embed_dim_]
 
-        return torch.cat((cls_token.expand(inputs.shape[0], -1, -1), inputs), dim=1)
+        if kwargs.get("mapping", False):
```

Comment:
Not necessary. It can always return a dict.

You only need to unpack a dict in `forward_with_args`.